bigstore/backends/rackspace.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from builtins import object
```

Comment:
Is this import necessary?

Reply:
In that case, let's leave it in there. Everything else looks great. So much for the contribution!